app/src/test/java/com/appunite/loudius/util/CoroutinesHelpers.kt

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+// chore: fix package name
```

Comment:
Will be fixed with https://loudius.atlassian.net/browse/LD-84?atlOrigin=eyJpIjoiZmRkY2E2ZDljNTUxNGQ0NGE5NmZlNDk5ZmM5MGIzYzEiLCJwIjoiaiJ9